test/recipes/02-test_errstr.t

```diff
@@ -69,6 +69,10 @@ foreach my $errname (@Errno::EXPORT_OK) {
       # is to skip this errcode.
       skip "perl error strings and ssystem error strings for errcode 0 differ", 1
           if $errcode == 0;
+      # On some systems, there are negative error codes.  These are currently
+      # unsupported in OpenSSL error reports.
```

Comment:
Would you mind to mention Hurd so it known which one caused the trouble?

Reply:
Done